test/deprecated-keyword/messageFiltering.chpl

```diff
@@ -106,6 +106,14 @@ deprecated "Lorem ipsum :proc:`:proc:` dolor sit amet (should not filter)"
 var x5011 = 5011;
 deprecated "Lorem ipsum :proc:`:proc:abc` dolor sit amet (should not filter)"
 var x5012 = 5012;
+deprecated "Lorem ipsum :proc:`title target` dolor sit amet (should not filter)"
+var x5013 = 5013;
+deprecated "Lorem ipsum :proc:`abc.def` dolor sit amet"
+var x5014 = 5014;
+deprecated "Lorem ipsum :proc:`~abc.def` dolor sit amet (should not filter)"
```

Comment:
As you've noted elsewhere, we want this to filter but it doesn't today.  I think it would be better for the test to mention that, right now it sounds like we don't intend for it to filter ever

Reply:
Sure, Ive separated a couple of those tests into their own section (saying they ideally shouldn't filter but currently do) and removed the "(should not filter)" text.

Reply:
Awesome, thanks!